### PR TITLE
[KYUUBI #3045][FOLLOWUP] Correct the common options and add docs for kyuubi-admin command

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

1. The version options should be in zkOptions.
2. Add docs for kyuubi-adminctl.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate
image

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: docs/tools/kyuubi-adminctl.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  --conf <value>           Kyuubi config property pair, formatted key=value.
+
+Command: refresh [config] <args>...
+        Refresh the resource.
+Command: refresh config [<configType>]
+        Refresh the config with specified type.
+  <configType>             The valid config type can be one of the following: hadoopConf.
+
+  -h, --help               Show help message and exit.
+```
```
Comment (yaooqinn) [inline docs/tools/kyuubi-adminctl.md:46]: remove these lines

File: docs/tools/kyuubi-adminctl.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# Administer kyuubi server Tool
```
Comment (yaooqinn) [inline docs/tools/kyuubi-adminctl.md:19]: capitalize all or just the leading word

File: docs/tools/kyuubi-adminctl.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
```
Comment (yaooqinn) [inline docs/tools/kyuubi-adminctl.md:20]: please describe what this tool is

Comment (yaooqinn) [reply docs/tools/kyuubi-adminctl.md:20]: as I don't understand why refreshing config is admin behavior but creating/deleting a server is not.

File: docs/tools/kyuubi-adminctl.md (hunk 1 above)
Comment (turboFei) [reply docs/tools/kyuubi-adminctl.md]: It refers the docs of kyuubi-ctl.

Comment (yaooqinn) [reply docs/tools/kyuubi-adminctl.md]: we are improving all the docs

File: docs/tools/kyuubi-adminctl.md (hunk 3 above)
Comment (turboFei) [reply docs/tools/kyuubi-adminctl.md:20]: can we move the server actions into kyuubi-adminctl later?

Comment (yaooqinn) [reply docs/tools/kyuubi-adminctl.md:20]: 1. kyuubi-admin looks better
2. > can we move the server actions into kyuubi-adminctl later?

we can but better duplicate the functions and deprecate them in ctl first

Comment (turboFei) [reply docs/tools/kyuubi-adminctl.md:20]: > 1. kyuubi-admin looks better
It sounds good and does not conflicts with kyuubi-ctl.

We can refactor the kyuubi-ctl in long term.

File: docs/tools/kyuubi-adminctl.md (hunk 1 above)
Comment (turboFei) [reply docs/tools/kyuubi-adminctl.md]: removed.

File: docs/tools/kyuubi-admin.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# Administer Kyuubi Server Tool
```
Comment (yaooqinn) [inline docs/tools/kyuubi-admin.md:19]: `Kyuubi Administer Tool`, as you plan to migrate operations with engines too.

File: docs/tools/kyuubi-admin.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+The tool to administer the Kyuubi server based on admin resource rest api.
+You can specify the rest host url(`--hostUrl`), auth schema(`--authSchema`), spnego host(`--spnegoHost`) and so on for rest rpc call.
```
Comment (yaooqinn) [inline docs/tools/kyuubi-admin.md:22]:
```suggestion
Kyuubi administer tool(kyuubi-admin) provides administrators with some maintenance operations against a kyuubi server or cluster.
```

File: docs/tools/kyuubi-admin.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
```
Comment (yaooqinn) [inline docs/tools/kyuubi-admin.md:23]:
```suggestion
## Installation
```

File: docs/tools/kyuubi-admin.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+## Usage
+```shell
+bin/kyuubi-admin --help
```
Comment (yaooqinn) [inline docs/tools/kyuubi-admin.md:26]: can we show help hint for sub commands?

**Comment by turboFei** [comment]
image

File: docs/tools/kyuubi-admin.md (hunk 7 above)
Comment (yaooqinn) [reply docs/tools/kyuubi-admin.md:43]: this is a question for the command, not the content you've written..

**Review by yaooqinn — APPROVED** [review]

**Comment by turboFei** [comment]
thanks @yaooqinn  merged to master

**Pull request closed without merging**